types/src/index.ts

```diff
@@ -1,25 +1,30 @@
 #!/usr/bin/env node
-import assert from "assert";
+import { readFileSync, readdirSync } from "fs";
 import { mkdir, readFile, readdir, writeFile } from "fs/promises";
+import assert from "node:assert";
```

Comment:
nit: given that this is an ESM, likely best to make all of the node built in specifiers use the `node:` prefix for consistency

Reply:
https://github.com/cloudflare/workerd/pull/2379/commits/409f10dca728a04a7800f20173a96aa75a4ae1b8